backend/project-service/src/main/kotlin/project/services/QueryService.kt

```diff
@@ -23,6 +23,28 @@ data class ProjectForVerification(
 class QueryService(
     private val projects: ProjectService
 ) {
+    suspend fun searchProjectPaths(ctx: DBContext, actor: Actor, query: String): Map<String, String> {
+        return ctx.withSession { session ->
```

Comment:
Missing authorization check. Not every user should be able to read this information. The user should be either a UCloud admin or member of the project itself.